### ISSUE TITLE
Highstock Intraday Volume column is invalid

### ISSUE THREAD

**Issue by gkislin** [highcharts/highcharts#359]
Hi!
I'm very plesend of your Highstock product.
But Intraday view volume columns for my data has a very thin invalid form. When I shift through the date it sometimes become normal.
This is html with js, in all (IE, FF, Chrome) browser this bug is happend:

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
<head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>Test</title>
</head>

<body>

<script type="text/javascript" language="javascript" src="ui/shared.js"></script>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.6.1/jquery.min.js" type="text/javascript"></script>

<script src="highstock/highstock.js" type="text/javascript"></script>

<script src="highstock/grid.js" type="text/javascript"></script>

<script src="highstock/exporting.js" type="text/javascript"></script>

<div id='gwt-uid-1' style="height: 500px"></div>

<script type="text/javascript">
    var chartParam={xAxis:{type:"datetime", maxZoom:1000}, yAxis:[{labels:{align:"left", x:2, y:-2}, showLastLabel:false, title:{text:"Closing"}, height:200, lineWidth:2},
    {labels:{align:"left", x:2, y:-2}, showLastLabel:false, title:{text:"Volume"}, top:300, height:100, offset:0, lineWidth:2}],
    rangeSelector:{buttons:[{type:"minute", count:1, text:"1m"},{type:"minute", count:15, text:"15m"},
    {type:"hour", count:1, text:"1h"},{type:"day", count:1, text:"1d"},{type:"all", count:1, text:"All"}],
    selected:0, inputEnabled:false}, chart:{renderTo:"gwt-uid-1"}, series:[
    {name:"PL", data:[[1313071951977,-81899.9],[1313071952497,-82000],[1313071952893,-82141.1],[1313071953177,-41000],[1313071953232,-41141.1],[1313071955360,-286699.6],[1313071955367,-327676.7],[1313071955607,-286676.7],[1313071956198,-245699.6],[1313071956500,-245817.8],[1313071956578,-245799.7],[1313071957250,-122599.6],[1313071957269,-204553.4],[1313071957331,-122576.7],[1313071957557,-81599.6],[1313071958487,-81699.7],[1313071958768,-81799.8],[1313071959120,-655594.5],[1313071959407,-614699.4],[1313071959927,-614594.5],[1313071960052,-573699.4],[1313071960367,-573799.5],[1313071960887,-573899.6],[1313071961127,-573753.4],[1313071961692,-491799.6],[1313071961899,-491653.4],[1313071962548,-409535.6],[1313071962628,-368599.5],[1313071962789,-368535.6],[1313071963042,-327599.5],[1313071963517,-327699.6],[1313071964311,-1065899.5],[1313071964337,-1065835.6],[1313071965087,-1024999.6],[1313071965202,-1024899.5],[1313071965596,-1024976.7],[1313071965792,-942899.5],[1313071965847,-983835.6],[1313071966288,-942999.6],[1313071966467,-942976.7],[1313071966527,-778453.4],[1313071966648,-901671.2],[1313071966705,-819594.5],[1313071966832,-737312.3],[1313071966893,-696171.2],[1313071967824,-655399.1],[1313071968380,-655499.2],[1313071968903,-655599.3],[1313071969854,-655699.4],[1313071970375,-1521335.6],[1313071970569,-1480399.5],[1313071971145,-1480384.4],[1313071971277,-1451599.5],[1313071971449,-1451535.6],[1313071971491,-1410599.5],[1313071971727,-1410535.6],[1313071972267,-1369599.5],[1313071972991,-1369535.6],[1313071972995,-1328394.5],[1313071973190,-1246399.3],[1313071973215,-1287253.4],[1313071973537,-1205399.3],[1313071973687,-1246253.4],[1313071973926,-1205499.4],[1313071974602,-1205189],[1313071974949,-1123399.4],[1313071975611,-1123355.5],[1313071975741,-1094599.4],[1313071976487,-1094699.5],[1313071976785,-1094684.4],[1313071977128,-1896113.3],[1313071977357,-1867299.6],[1313071978059,-1867399.7],[1313071978359,-1867442.2],[1313071978398,-1838599.7],[1313071979208,-1838699.8],[1313071979547,-1838799.9],[1313071979717,-1838900],[1313071979917,-1809942.2],[1313071980799,-1752199.8],[1313071981940,-1752271.1],[1313071982009,-1723399.8],[1313071982093,-1694542.2],[1313071982218,-1723471.1],[1313071982497,-1665613.3],[1313071982961,-1636799.6],[1313071984182,-1636899.7],[1313071984912,-1579113.3],[1313071985034,-1636884.4],[1313071985315,-1550184.4],[1313071985724,-1521399.5],[1313071986310,-2495699.5],[1313071986627,-2495799.6],[1313071988369,-2495899.7],[1313071989108,-2495600],[1313071989270,-2295999.7],[1313071989977,-2296099.8],[1313071990735,-2296000],[1313071991607,-2096099.7],[1313071991800,-2096199.8],[1313071992724,-2095999.6],[1313071992931,-2096099.7],[1313071994197,-2095999.6],[1313071994935,-2095500],[1313071995246,-1896099.6],[1313071995545,-2580700.1],[1313071995576,-2781100],[1313071996267,-2380599.9],[1313071996378,-2580600],[1313071996532,-2380700],[1313071996937,-2580600],[1313071997037,-2380400],[1313071997214,-2580458.9],[1313071997616,-2539313],[1313071999932,-2495899.7],[1313072001505,-2495799.6],[1313072003637,-2707500.1],[1313072003964,-2707746.6],[1313072004099,-2634300.1],[1313072004427,-2634400.2],[1313072005323,-2634300.1],[1313072005780,-2634500.3],[1313072005789,-2634400.2],[1313072006337,-2634800],[1313072006345,-2834400.3],[1313072006493,-2834500.4],[1313072006887,-2834480.7],[1313072007117,-2780600.1],[1313072007207,-2807553.8],[1313072007318,-2871399.7],[1313072007727,-2871499.8],[1313072007787,-2871399.7],[1313072007974,-2871299.6],[1313072008087,-2871499.8],[1313072008364,-2871399.7],[1313072008395,-2871399.7],[1313072008553,-2871499.8],[1313072008746,-2871399.7],[1313072009650,-2871599.9],[1313072009695,-2871499.8],[1313072010463,-2871964.4],[1313072010507,-2871700],[1313072011024,-2707400],[1313072011801,-2707299.9],[1313072012169,-2707400],[1313072012869,-3166099.5],[1313072013215,-3165838.6],[1313072013309,-3139099.3],[1313072014171,-3139299.5],[1313072014191,-3139199.4],[1313072014537,-3139292.4],[1313072015291,-3112365.5],[1313072016003,-3058799.4],[1313072016004,-3058699.3],[1313072016287,-3085438.6],[1313072016538,-3058899.5],[1313072016977,-3058999.6],[1313072017147,-3059019.3],[1313072017425,-3032199.6],[1313072017829,-3032299.7],[1313072018529,-3032346.2],[1313072018725,-3005499.7],[1313072019827,-3005546.2],[1313072019997,-2978699.7],[1313072020387,-2978746.2],[1313072020721,-2951819.3],[1313072020917,-2924999.6],[1313072021877,-2871299.6],[1313072021886,-2924938.6],[1313072022237,-3514492.4],[1313072022412,-3487565.5],[1313072022608,-3433899.3],[1313072022661,-3460638.6],[1313072023159,-3433999.4],[1313072023467,-3433965.5],[1313072024027,-3380299.3],[1313072024261,-3407038.6],[1313072025262,-3380399.4],[1313072025782,-3380499.5],[1313072026181,-3380599.6],[1313072026949,-3380699.7],[1313072027446,-3380599.6],[1313072028807,-3380619.3],[1313072029020,-3353799.6],[1313072030379,-3300099.5],[1313072030447,-3326892.4],[1313072030463,-3353819.3],[1313072030687,-3300092.4],[1313072030931,-3273165.5],[1313072031737,-3246238.6],[1313072031761,-3219311.7],[1313072031922,-3192599.2],[1313072032383,-3192511.7],[1313072032797,-3165799.2],[1313072033225,-3165899.3],[1313072033652,-3165999.4],[1313072034666,-3891638.6],[1313072034980,-3864899.3],[1313072035417,-3864999.4],[1313072035649,-3864831],[1313072035865,-3811138.6],[1313072036287,-3784399.3],[1313072037307,-3784499.4],[1313072038378,-3784599.5],[1313072038657,-3784592.4],[1313072039335,-3757799.5],[1313072039728,-3757792.4],[1313072040357,-3730999.5],[1313072040871,-3730300],[1313072040874,-3531099.5],[1313072041177,-3531092.4],[1313072042232,-3504299.5],[1313072044087,-3477499.5],[1313072044166,-3504292.4],[1313072044578,-3477599.6],[1313072045525,-3477699.7],[1313072045985,-3477799.8],[1313072046087,-3477100],[1313072046268,-3650699.6],[1313072046324,-3677519.3],[1313072046350,-3677699.8],[1313072046572,-3650719.3],[1313072046824,-3623899.6],[1313072047003,-3623919.3],[1313072047798,-3597099.6],[1313072048017,-3596731],[1313072048210,-3543199.4],[1313072048617,-3543299.5],[1313072048813,-3543284.4],[1313072049503,-3514499.5],[1313072066375,-4144200],[1313072067248,-3945399.3],[1313072067771,-3945499.4],[1313072068289,-3945599.5],[1313072068352,-3945177.2],[1313072068786,-3891699.3]]},
    {name:"volume", type:"column", yAxis:1, data:[[1313071951977,100],[1313071952497,100],[1313071952893,-41100],[1313071953177,100],[1313071953232,-41100],[1313071955360,100],[1313071955367,-41100],[1313071955607,-41100],[1313071956198,100],[1313071956500,-41100],[1313071956578,100],[1313071957250,100],[1313071957269,-82200],[1313071957331,-41100],[1313071957557,100],[1313071958487,100],[1313071958768,100],[1313071959120,-41100],[1313071959407,100],[1313071959927,-41100],[1313071960052,100],[1313071960367,100],[1313071960887,100],[1313071961127,-82200],[1313071961692,100],[1313071961899,-82200],[1313071962548,-41100],[1313071962628,100],[1313071962789,-41100],[1313071963042,100],[1313071963517,100],[1313071964311,100],[1313071964337,-41100],[1313071965087,100],[1313071965202,100],[1313071965596,-41100],[1313071965792,100],[1313071965847,-41100],[1313071966288,100],[1313071966467,-41100],[1313071966527,-41100],[1313071966648,-82200],[1313071966705,-41100],[1313071966832,-41100],[1313071966893,-41100],[1313071967824,100],[1313071968380,100],[1313071968903,100],[1313071969854,100],[1313071970375,-41100],[1313071970569,100],[1313071971145,-28900],[1313071971277,100],[1313071971449,-41100],[1313071971491,100],[1313071971727,-41100],[1313071972267,100],[1313071972991,-41100],[1313071972995,-41100],[1313071973190,100],[1313071973215,-41100],[1313071973537,100],[1313071973687,-41100],[1313071973926,100],[1313071974602,-82200],[1313071974949,100],[1313071975611,-28900],[1313071975741,100],[1313071976487,100],[1313071976785,-28900],[1313071977128,-28900],[1313071977357,100],[1313071978059,100],[1313071978359,-28900],[1313071978398,100],[1313071979208,100],[1313071979547,100],[1313071979717,-28900],[1313071979917,-57800],[1313071980799,100],[1313071981940,-28900],[1313071982009,100],[1313071982093,-28900],[1313071982218,-28900],[1313071982497,-28900],[1313071982961,100],[1313071984182,100],[1313071984912,-28900],[1313071985034,-57800],[1313071985315,-28900],[1313071985724,100],[1313071986310,100],[1313071986627,100],[1313071988369,100],[1313071989108,-200000],[1313071989270,100],[1313071989977,100],[1313071990735,-200000],[1313071991607,-100],[1313071991800,-100],[1313071992724,100],[1313071992931,-100],[1313071994197,100],[1313071994935,-200000],[1313071995246,100],[1313071995545,-100],[1313071995576,-200000],[1313071996267,100],[1313071996378,-200000],[1313071996532,-100],[1313071996937,-100],[1313071997037,200000],[1313071997214,-41100],[1313071997616,-43500],[1313071999932,-100],[1313072001505,-100],[1313072003637,100],[1313072003964,-73300],[1313072004099,100],[1313072004427,-100],[1313072005323,100],[1313072005780,-100],[1313072005789,100],[1313072006337,200000],[1313072006345,100],[1313072006493,-100],[1313072006887,-26900],[1313072007117,100],[1313072007207,-26900],[1313072007318,100],[1313072007727,-100],[1313072007787,-100],[1313072007974,100],[1313072008087,-100],[1313072008364,100],[1313072008395,100],[1313072008553,-100],[1313072008746,100],[1313072009650,100],[1313072009695,100],[1313072010463,-164400],[1313072010507,100],[1313072011024,-100],[1313072011801,100],[1313072012169,100],[1313072012869,-100],[1313072013215,-26900],[1313072013309,100],[1313072014171,100],[1313072014191,100],[1313072014537,-26900],[1313072015291,-26900],[1313072016003,100],[1313072016004,100],[1313072016287,-26900],[1313072016538,100],[1313072016977,100],[1313072017147,-26900],[1313072017425,100],[1313072017829,100],[1313072018529,-26900],[1313072018725,100],[1313072019827,-26900],[1313072019997,100],[1313072020387,-26900],[1313072020721,-26900],[1313072020917,100],[1313072021877,100],[1313072021886,-53800],[1313072022237,-26900],[1313072022412,-26900],[1313072022608,100],[1313072022661,-26900],[1313072023159,100],[1313072023467,-26900],[1313072024027,100],[1313072024261,-26900],[1313072025262,100],[1313072025782,100],[1313072026181,100],[1313072026949,-100],[1313072027446,100],[1313072028807,-26900],[1313072029020,100],[1313072030379,100],[1313072030447,-26900],[1313072030463,-26900],[1313072030687,-26900],[1313072030931,-26900],[1313072031737,-26900],[1313072031761,-26900],[1313072031922,100],[1313072032383,-26900],[1313072032797,100],[1313072033225,100],[1313072033652,100],[1313072034666,-26900],[1313072034980,100],[1313072035417,100],[1313072035649,-53800],[1313072035865,-26900],[1313072036287,100],[1313072037307,100],[1313072038378,100],[1313072038657,-26900],[1313072039335,100],[1313072039728,-26900],[1313072040357,100],[1313072040871,-200000],[1313072040874,100],[1313072041177,-26900],[1313072042232,100],[1313072044087,100],[1313072044166,-26900],[1313072044578,100],[1313072045525,100],[1313072045985,-100],[1313072046087,200000],[1313072046268,100],[1313072046324,-26900],[1313072046350,-100],[1313072046572,-26900],[1313072046824,100],[1313072047003,-26900],[1313072047798,-100],[1313072048017,-53800],[1313072048210,100],[1313072048617,100],[1313072048813,-28900],[1313072049503,100],[1313072066375,-200000],[1313072067248,100],[1313072067771,100],[1313072068289,-100],[1313072068352,-53800],[1313072068786,100]]}]}
    new Highcharts.StockChart(chartParam);
</script>

</body>
</html>
